src/loader.js

```diff
@@ -284,6 +284,85 @@ function setupModuleLoader(window) {
 
           /**
            * @ngdoc method
+           * @name angular.Module#component
+           * @module ng
+           * @param {string} name Name of the component in camel-case (i.e. myComp which will match as my-comp)
+           * @param {Object} options Component definition object, has the following properties (all optional):
```

Comment:
Or perhaps we should generally start these descriptions with an upper case character ! (Some of them are, some aren't .)
It makes it much more readable (in code) and I am happy when I see an uppercased description :)